govcd/api_vcd_test.go

```diff
@@ -1033,6 +1033,24 @@ func (vcd *TestVCD) removeLeftoverEntities(entity CleanupEntity) {
 			vcd.infoCleanup(notDeletedMsg, entity.EntityType, entity.Name, err)
 		}
 		return
+	case "standaloneVm":
+		vm, err := vcd.vdc.QueryVmById(entity.Name) // The VM ID must be passed as Name
+		if IsNotFound(err) {
+			vcd.infoCleanup(notFoundMsg, entity.EntityType, entity.Name)
+			return
+		}
+		if err != nil {
+			vcd.infoCleanup("removeLeftoverEntries: [ERROR] retrieving standalone VM '%s''. %s\n",
```

Comment:
There is one too many apostrophe in the string.

Reply:
fixed